cmd/listClients_test.go

```
@@ -0,0 +1,21 @@
+package cmd
+
+import (
+	"bytes"
+	"testing"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func TestListClientsCmd(t *testing.T) {
```

This should be tested with several configuration files, bc it depends on a config file. It should throw an error if clients are not listed correctly on the config file

---

Yes, I will add them

---

I will add different configuration files to the tests data. But still the problem with the program exiting from the test using `log.Fatal(err)` in case of errors exists.